src/Sarif.Driver.UnitTests/ExceptionRaisingRule.cs

```diff
@@ -105,12 +105,19 @@ public void Analyze(TestAnalysisContext context)
 
             if (_exceptionCondition == ExceptionCondition.ParsingTarget)
             {
-                Errors.LogTargetParseError(context, null, "Could not parse target.");
+                Errors.LogTargetParseError(
+                    context,
+                    new Region
+                    {
+                        StartLine = 42,
+                        StartColumn = 54
+                    },
+                    "Could not parse target.");
```

Comment:
Filed [https://github.com/Microsoft/sarif-sdk/issues/232](https://github.com/Microsoft/sarif-sdk/issues/232)


---
In reply to: [61635832](https://github.com/Microsoft/sarif-sdk/pull/229#discussion_r61635832) [](ancestors = 61635832,61528567)